_posts/2019-10-28-upgrading-a-fairly-large-rails-application .markdown

```diff
@@ -0,0 +1,24 @@
+layout: post
+title:  "Upgrading a Fairly Large Rails Application from 4.2 to 5.2"
+date: 2019-10-28 15:00:00
+categories: ["rails", "upgrades", "case-study"]
+author: abizzinotto
+---
+
+We recently collaborated with [Amitree](https://www.home.amitree.com/) on a Rails upgrade for their application which "helps real estate agents organize their email and keep their clients up to date on their transactions" and were able to speak to them about their experience with Ombu Labs.
+
+<!--more-->
+
+We spoke with [Marcus McLaughlin](https://www.linkedin.com/in/marcusmclaughlin/), Head of Engineering at [Amitree](https://www.home.amitree.com/), about the work performed to upgrade their app. Marcus states that their application is fairly large, about 5 year old and has a heavy reliance on background processing.
+
+[Amitree](https://www.home.amitree.com/) was having difficulty allocating developer attention to the Rails upgrade project, a common issue experienced by many companies. Their application fell out of date because, in Marcus' words, "upgrading continually fell down our priority list, and we never had a business critical reason to upgrade the application until the version we were running was about to go entirely out of support". That is why they decided to come to Ombu Labs for help.
+
+Initially, [Amitree](https://www.home.amitree.com/) ordered the [OmbuLabs Roadmap](https://www.fastruby.io/roadmap) so they could have a good estimate of what would be needed to do the upgrade. The roadmap proved to be useful "both in terms of the information that it provided, but also in terms of building my [Marcus'] confidence in Ombu Labs ability to deliver the estimate".
+
+We then executed a full upgrade of the application from Rails 4.2 to 5.2. This necessary upgrade of their application ensured their application continued running on a Rails version that is still supported, preventing security issues and bugs.
+
+According to Marcus, the Ombu Labs team was responsive and self-sufficient, which resulted in the time the [Amitree](https://www.home.amitree.com/) team needed to dedicate to the upgrade being "considerably less than I [Marcus] would have expected". Ombu Labs' exclusive focus on the Rails upgrade allowed the [Amitree](https://www.home.amitree.com/) team to continue focusing on their work to achieve their other business goals and, in the end, they had a fully upgraded Rails application and had "zero major production issues as a result of the upgrade process, and also saw very little decrease in productivity as a team during the upgrade."
```

Comment:
@ABizzinotto Could you break the last sentence into multiple ones? It is hard to follow such a long sentence.